modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/SequencesOnDisplay.scala

```diff
@@ -84,73 +84,76 @@ final case class SequencesOnDisplay(tabs: Zipper[SeqexecTab]) {
     */
   def updateFromQueue(s: SequencesQueue[SequenceView]): SequencesOnDisplay = {
     // Sequences in view stored after completion
-    val cids: List[Option[SequenceView]] = SequencesOnDisplay.completedTabs
-      .getAll(this)
-      .filterNot(x => s.loaded.values.toList.contains(x.id))
-      .map(_.some)
-    val updated = updateLoaded(s.loaded.values.toList.map { id =>
+    val completedIds: List[Option[SequenceView]] =
+      SequencesOnDisplay.completedTabs
+        .getAll(this)
+        .filterNot(x => s.loaded.values.toList.contains(x.id))
+        .map(_.some)
+    val allIds = s.sessionQueue.map(_.id)
+    val loadedInSessionQueue = s.loaded.values.toList.map { id =>
       s.sessionQueue.find(_.id === id)
```

Comment:
I guess here you could now work with `allIds` instead? Not really an issue but just an observation. (Pun not intended.)

Reply:
😄